src/eos_stream.c

```diff
@@ -356,11 +367,17 @@ void printArgument(uint8_t argNum, txProcessingContext_t *context) {
 static bool isKnownAction(txProcessingContext_t *context) {
     name_t contractName = context->contractName;
     name_t actionName = context->contractActionName;
-    if (actionName == EOSIO_TOKEN_TRANSFER) {
+    
+    if (actionName == VAULTA_SWAPTO && contractName == CORE_VAULTA) {
```

Comment:
You could add `&& isTransferDataValid(context->currentFieldLength)` here as well.